src/coordinatorapi.h

```diff
@@ -79,9 +79,6 @@ char *connectAndSendUserCommand(char c,
                                 int *isRunning = NULL,
                                 int *ckptInterval = NULL);
 
-void updateCoordCkptDir(const char *dir);
-string getCoordCkptDir(void);
-
```

Comment:
As before, any discussion of removing some functionality should be a separate PR.